encodings/fastlanes/src/bitpacking/mod.rs

```diff
@@ -127,6 +127,11 @@ impl BitPackedArray {
     pub fn ptype(&self) -> PType {
         self.dtype().try_into().unwrap()
     }
+
+    #[inline]
+    pub fn max_packed_value(&self) -> usize {
+        1 << self.bit_width()
```

Comment:
shouldn't this also account for the offset, if nonzero?

Reply:
No, offset is the logical offset into the underlying array not the size that the array has been packed into.